code/game/machinery/anomaly_pad.dm

```diff
@@ -0,0 +1,124 @@
+///Magical pad that suspends anomalies. Simply apply an anomaly core and it will make the anomaly fly above it
+/obj/machinery/anomaly_pad
+	name = "anomaly pad"
+	desc = "If an anomaly core is applied, this machine will keep it stable and suspended. Looks beautiful and cannot possibly go wrong."
+
+	icon = 'icons/obj/machines/anomaly_pad.dmi'
+	icon_state = "pad"
+
+	circuit = /obj/item/circuitboard/machine/anomaly_pad
+	density = TRUE
+
+	use_power = IDLE_POWER_USE
+	power_channel = EQUIP
+
+	///this is the big beautiful anomaly floating ontop of us
+	var/obj/effect/anomaly/anomaly
+	///our base active power use. Decreases with laser efficiency and is * 0.1 for inactive
+	var/base_power = 500
+
+/obj/machinery/anomaly_pad/RefreshParts()
+	var/obj/item/stock_parts/micro_laser/L = locate(/obj/item/stock_parts/micro_laser) in contents
+
+	if(L)
+		active_power_usage = base_power / L.rating
+		idle_power_usage = active_power_usage * 0.1
+
+/obj/machinery/anomaly_pad/attackby(obj/item/I, mob/living/user, params)
+
+	if(default_deconstruction_screwdriver(user, icon_state, icon_state, I))
+		update_icon_state()
+		return
+
+	else if(default_deconstruction_crowbar(I))
+		return
+
+	if(!anomaly && istype(I, /obj/item/assembly/signaler/anomaly) && user.dropItemToGround(I) && IsAnomalyApplicable(I, user))
+		CaptureAnomaly(I)
+		playsound(loc, 'sound/machines/click.ogg', 15, TRUE)
+		return
+
+	return ..()
+
+/obj/machinery/anomaly_pad/process()
+	if(anomaly && (!is_operational() || get_turf(src) != get_turf(anomaly)))
+		ReleaseAnomaly()
+		return FALSE //uuhhh dont know if its ok but im using this to see if the check passed on the subtype
+
+	return TRUE
```

Comment:
The return here is only relevant if it is `PROCESS_KILL`, in where it stops processing.
Could you make it start processing on `CaptureAnomaly()` and stop on `ReleaseAnomaly()`?